source/concepts/flakes.md

```diff
@@ -25,7 +29,7 @@ Flakes add the following behavior to Nix:
 
    [new command line interface]: https://nix.dev/manual/nix/2.18/command-ref/new-cli/nix.html
 
-Nix handles flakes differently than regular {term}`Nix file`s in the following ways:
+Nix handles flakes differently than regular {term}`Nix files <Nix file>` in the following ways:
 
 - The `flake.nix` file is checked for schema validity.
```

Comment:
```suggestion
- The `flake.nix` file is checked for schema validity with the [nix flake check](https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-flake-check) command.
```

Reply:
It's always checked. Try running `nix build` on an arbitrary Nix expression.